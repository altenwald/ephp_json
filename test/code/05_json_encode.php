<?php

var_dump(json_encode(12.01));
var_dump(json_encode(12.0, JSON_PRESERVE_ZERO_FRACTION));
var_dump(json_encode(12.0));
