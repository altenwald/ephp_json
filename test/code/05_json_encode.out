string(7) "1.2e+01"
string(4) "12.0"
string(2) "12"
